app/Providers/AppServiceProvider.php

```diff
@@ -20,6 +22,8 @@ public function boot()
         if (env('APP_DEBUG')) {
             $this->register(Debugbar\ServiceProvider::class);
         }
+        // Add obersations class
+        Post::observe(PostsCountObserver::class);
```

Comment:
Je placerais les observer dans un fichier ObserverServiceProvider afin de tous les centraliser
`<?php namespace App\Providers;

use Illuminate\Support\ServiceProvider;

class ObserverServiceProvider extends ServiceProvider
{

  /**
   * Bootstrap any necessary services.
   *
   * @return void
   */
  public function boot()
  {
    \App\User::observe( new \App\Observers\UserObserver );
  }

  /**
   * Register the service provider.
   *
   * @return void
   */
  public function register()
  {
  }
}`

Et ajouté ce provider dans le config/app.php

Reply:
Ha oui carrément, ça ne me plaisait pas trop on plus de le mettre là.

Reply:
C'est fait.